ui/src/components/view/ListResourceTable.vue

```diff
@@ -50,6 +50,10 @@
             {{ $toLocaleDate(text) }}
           </template>
 
+          <template v-else-if="column.key === 'size' || column.key === 'virtualsize'">
+            {{ parseFloat(text / (1024.0 * 1024.0 * 1024.0)).toFixed(2) }} GB
+          </template>
+
```

Comment:
would it be better to use - `bytesToHumanReadableSize` method?

Reply:
@Pearl1594 to check & advise.